docs/local-development.md

```diff
@@ -0,0 +1,74 @@
+# Local Development
+
+## Pre-reqs
+
+* Access to an Openshift cluster
+* A clone of [korekuta-go-operator](https://github.com/project-koku/korekuta-operator-go)
+* [Go 1.13 or greater](https://golang.org/doc/install)
+* [Openshift-CLI](https://docs.openshift.com/container-platform/4.5/cli_reference/openshift_cli/getting-started-cli.html) (preferably a version that matches your Openshift cluster version)
+* [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/)
+* [kustomize](https://kubernetes-sigs.github.io/kustomize/installation/) (before installing this separately, check that it was not already installed along with kubectl)
+* [Docker Desktop](https://www.docker.com/products/docker-desktop)
+* quay.io account
+
+## Running the operator locally
+
+1. Log into your OCP cluster from a terminal, create an `openshift-cost` namespace, and switch to the new namespace:
+
+    ```
+    $ oc login --token=<token> --server=<server>
+    $ oc create namespace openshift-cost
+    $ oc project openshift-cost
+    ```
+
+2. Build the manager binary:
+
+    ```
+    $ make manager
+    ```
+
+3. Register the CRD with the Kubernetes apiserver:
+
+    ```
+    $ make install
+    ```
+
+4. Deploy the operator
+
+    ```
+    make run ENABLE_WEBHOOKS=false
+    ```
+
+    At this point, you will see the operator spin up in your terminal. After a few seconds, you should see something similar to the following output:
+    ```
+    2020-10-21T09:31:37.195-0400    INFO    controller-runtime.controller   Starting workers        {"controller": "costmanagement", "worker count": 1}
+    ```
+    The operator is running but is not doing any work. We need to create a CR.
+
+5. Deploy a CR. For now, use basic authentication. The following creates the appropriate spec. `username` and `password` correspond to the username (not email address) and password for the account you want to use at cloud.redhat.com:
```

Comment:
We should document that if you just run `make deploy-cr` it will use token authentication.